mkosi/resources/man/mkosi.1.md

```diff
@@ -1195,6 +1195,11 @@ boolean argument: either `1`, `yes`, or `true` to enable, or `0`, `no`,
 `Sign=`, `--sign`
 :   Sign the generated `SHA256SUMS` using `gpg` after completion.
 
+`OpenpgpTool=`, `--openpgp-tool`
+:   OpenPGP implementation to use for signing. `gpg` is the default.
+    Selecting a value different than the default will use a given Stateless
```

Comment:
```suggestion
    Selecting a value different than the default will use the given Stateless
```
Also, itt would be nice to add an example here, e.g. `sqop`.

Reply:
Good idea with the example. I could list two (say `sqop` and `rsop`). What do you think about linking to https://www.openpgp.org/about/sop/ (which lists an even wider roster of implementations)? Would that be a bit too much?

Reply:
I don't think that would be too much, how about
```suggestion
    OpenPGP (SOP) tool for signing the `SHA256SUMS` file.
    
    Examplary choices are `sqop` and `rsop`, but any implementation from 
    https://www.openpgp.org/about/sop/ that can be installed locally will work
```

Reply:
Excellent wording, thanks for the suggestion! I just appended a dot at the end and re-pushed. I've also checked it locally with `./tools/make-man-page.sh` and `man ./mkosi/resources/man/mkosi.1`. It looked fine.